src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -109,70 +108,42 @@ function trackValueOnNode(node: any, inst: any): ?ValueTracker {
 
 var inputValueTracking = {
   // exposed for testing
-  _getTrackerFromNode(node: ElementWithWrapperState) {
-    return getTracker(ReactDOMComponentTree.getInstanceFromNode(node));
+  _getTrackerFromNode(inst: ElementWithValueTracker) {
```

Comment:
This seems like it should be `node`. We can also just pass it directly: `_getTrackerFromNode: getTracker`